Assets/VRTK/Scripts/VRTK_InteractTouch.cs

```diff
@@ -401,6 +401,7 @@ private void CreateTouchCollider()
                 {
                     controllerCollisionDetector = Instantiate(customRigidbodyObject, transform.position, transform.rotation) as GameObject;
                     controllerCollisionDetector.transform.SetParent(transform);
+                    controllerCollisionDetector.transform.localScale = transform.localScale;
```

Comment:
if parenting rescales to match, is there a nee to specifically set the localScale to match the parent?